lib/plotting_functions.py

```diff
@@ -134,519 +94,11 @@ def my_formatwarning(msg, *args, **kwargs):
-                assert ((12 in data.shape) and (data.shape.count(12) == 1)), f"Sorry, {data.shape.count(12)} dimensions have size 12, making determination of which dimension is month ambiguous. Please provide a `time` or `month` dimension."
-                time_dim_num = data.shape.index(12)
-                fakedims = [f"dim{n}" for n in range(len(data.shape))]
-                fakedims[time_dim_num] = "time"
-                data = xr.DataArray(data, dims=fakedims, attrs=data.attrs)
-            timefix = pd.date_range(start='1/1/1999', end='12/1/1999', freq='MS') # generic time coordinate from a non-leap-year
-            data = data.assign_coords({"time":timefix})
-        month_length = data.time.dt.days_in_month
-    #End try/except
-
-    data = data.sel(time=data.time.dt.month.isin(seasons[season])) # directly take the months we want based on season kwarg
-    return data.weighted(data.time.dt.daysinmonth).mean(dim='time', keep_attrs=True)
-
-
-
-#######
-
-#Polar Plot functions
-
-def domain_stats(data, domain):
-    """Provides statistics in specified region.
-
-    Parameters
-    ----------
-    data : xarray.DataArray
-        data values
-    domain : list or tuple or numpy.ndarray
-        the domain specification as:
-        [west_longitude, east_longitude, south_latitude, north_latitude]
-
-    Returns
-    -------
-    x_region_mean : float
-        the regional area-weighted average
-    x_region_max : float
-        the maximum value in the region
-    x_region_min : float
-        the minimum value in the region
-
-    Notes
-    -----
-    Currently assumes 'lat' is a dimension and uses `cos(lat)` as weight.
-    Should use `spatial_average`
-
-    See Also
-    --------
-    spatial_average
-
-    """
-    x_region = data.sel(lat=slice(domain[2],domain[3]), lon=slice(domain[0],domain[1]))
-    x_region_mean = x_region.weighted(np.cos(np.deg2rad(x_region['lat']))).mean().item()
-    x_region_min = x_region.min().item()
-    x_region_max = x_region.max().item()
-    return x_region_mean, x_region_max, x_region_min
-
-def make_polar_plot(wks, case_nickname, 
-                    base_nickname,
-                    case_climo_yrs, 
-                    baseline_climo_yrs,
-                    d1:xr.DataArray, 
-                    d2:xr.DataArray, 
-                    difference:Optional[xr.DataArray]=None,
-                    pctchange:Optional[xr.DataArray]=None,
-                    domain:Optional[list]=None, 
-                    hemisphere:Optional[str]=None, 
-                    obs=False, 
-                    **kwargs):
+def make_polar_plot(adfobj, wks, case_nickname, base_nickname,
+                    case_climo_yrs, baseline_climo_yrs,
+                    d1:xr.DataArray, d2:xr.DataArray, difference:Optional[xr.DataArray]=None,pctchange:Optional[xr.DataArray]=None,
+                    domain:Optional[list]=None, hemisphere:Optional[str]=None, obs=False, **kwargs):
```

Comment:
Was there a reason the argument spacing here was changed?  If not then I might request changing it back to a "one argument per line" style like it was previously.